lib/puppet/provider/selinux_port/semanage.rb

```diff
@@ -5,11 +5,13 @@
   # SELinux must be enabled. Is there a way to get a better error message?
   confine selinux: true
 
+  # custom fact, needed for fedora 24+
+  python_command = Facter.value(:semanage_is_python3) ? 'python3' : 'python'
```

Comment:
both python and python3 will be available, but only the other one might have the library needed, so I need a fact to detect this.

With this change, the acceptance tests seem to pass on everything relevant.